src/main/java/uk/gov/companieshouse/lookup/controller/GlobalController.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.lookup.controller;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.servlet.LocaleResolver;
+
+@ControllerAdvice
+public class GlobalController {
+    @Autowired
+    private LocaleResolver localeResolver;
+
```

Comment:
Where are these modelattributes queried/used?

Reply:
Ignore this comment as I found the info here: https://github.com/companieshouse/common-web-java/blob/983c8312b5221673a1b531b9c04288aa21123d61/README.md?plain=1#L136